### PR TITLE
Handle backslash-prefixed trigger_error calls

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
When using e.g. [this sniff](https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesfullyqualifiedglobalfunctions-), the `@\trigger_error` will lead to errors.

**Comment by mmoll** [comment]
@xalopp @Khartir could you have a look? :eyes:

File: Symfony/Sniffs/Errors/UserDeprecatedSniff.php [hunk 1]
```diff
@@ -74,7 +74,9 @@ public function process(File $phpcsFile, $stackPtr)
             }
 
             if ($tokens[$string]['content'] === 'E_USER_DEPRECATED'
-                && '@' !== $tokens[$stackPtr -1]['content']
+                && !('@' === $tokens[$stackPtr - 1]['content']
+                || ('@' === $tokens[$stackPtr - 2]['content']
+                && 'T_NS_SEPARATOR' === $tokens[$stackPtr - 1]['type']))
```
[the remark this answers is not in the document]
Comment (mmoll) [reply Symfony/Sniffs/Errors/UserDeprecatedSniff.php]: sure thing, changed. :+1:

**Comment by mmoll** [comment]
@djoos WDYT?

**Comment by djoos** [comment]
Thanks for your contribution @mmoll and @Khartir and @xalopp for the reviews!

**Pull request merged**